### ISSUE TITLE
`setup-middleware-reporter.d.ts` is causing typescript errors in consuming apps when updating to v5.2.0

### ISSUE THREAD

**Issue by robostheimer** [ember-a11y/ember-a11y-testing#502]
My team recently updated ember-a11y-testing to v 5.2.0 and we began to see typescript errors emanating from `ember-a11y-testing/test-support/setup-middleware-reporter.d.ts`

```node_modules/ember-a11y-testing/test-support/setup-middleware-reporter.d.ts:1:23 - error TS2688: Cannot find type definition file for 'ember__test-helpers'.```

```1 /// <reference types="ember__test-helpers" />```

In our tests, rolling back to v5.1.0 solves the issue.  

It appears to have something to do with the use of `currentRouteName` in https://github.com/ember-a11y/ember-a11y-testing/compare/v5.1.0...v5.2.0#diff-2b7b86d547afb814ffca762b5a1162f99ed0f3fbf12e23bd59ed82343d6270e4R48

**Comment by chriskrycho** [comment]
Thanks for reporting this! This package needs to update to use the official types from that package rather than [shipping its own](https://github.com/ember-a11y/ember-a11y-testing/blob/master/types/%40ember/test-helpers.d.ts)!

**Comment by robostheimer** [comment]
That's what we thought was going on given that `ember__test-helpers` is deprecated and essentially just a placeholder.

**Comment by robostheimer** [comment]
Wondering if we can get an update on this.  Starting to block a migration I'm working on.

**Comment by robostheimer** [comment]
Note removing the comment for `/// <reference types="ember_test-helpers />` from setup-middleware-reporter.d.ts` fixes the issue.

**Comment by robostheimer** [comment]
PR to fix: https://github.com/ember-a11y/ember-a11y-testing/pull/503

**Comment by drewlee** [comment]
This is pretty much a duplicate of issue #498.

**Comment by robostheimer** [comment]
This issue and likely #498 are fixed now.  Closing this issue now.
